hibernate-ucp/src/test/resources/hibernate.properties

```diff
@@ -0,0 +1,12 @@
+hibernate.dialect @db.dialect@
+hibernate.connection.driver_class @jdbc.driver@
+hibernate.connection.url @jdbc.url@
+hibernate.connection.username @jdbc.user@
+hibernate.connection.password @jdbc.pass@
+hibernate.connection.init_sql @connection.init_sql@
+hibernate.connection.provider_class=UCPConnectionProvider
+
+hibernate.oracleucp.connectionFactoryClassName=org.h2.jdbcx.JdbcDataSource
```

Comment:
@Sreekara-Reddy Search for a template, something like @jdbc.datasource@, instead of according the h2 datasource.